app/migrations/20231126065143_create_initial_tables.sql

```diff
@@ -0,0 +1,6 @@
+-- migrations/20230329_123456_create_initial_tables.sql
+CREATE TABLE my_new_table (
```

Comment:
Should we make this a `users` table or something we might actually use?

Reply:
yeah so im just getting the foundation set up for this, we can do a migration once we get a schema going